framework/src/messaging/Kconfig

```diff
@@ -0,0 +1,39 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config MESSAGING_IPC
+	bool "Enable Messaging APIs"
+	default y
+	depends on !DISABLE_MQUEUE
+	depends on !DISABLE_SIGNALS
+	---help---
+		Enables Messaging APIs.
+		Users can send and receive the data with unicast messaging IPC.
+		unicast messaging IPC supports like below.
+		send : sync/async mode
+			- sync : sender waits after sending msg
+			- async : sender just sends msg and be finished.
+		receive : block/non-block mode
+			- block : receiver waits for getting msg
+			- non-block : receiver sets msg callback func and continue to run.
+		reply : send reply msg from receiver to sender when sender sends sync msg
+			- using same port name for sending reply
+
+
+if MESSAGING_IPC
+config MESSAGING_UNICAST_MAXMSG
+	int "Max message number for unicast"
+	default 100
+	---help---
+		Max number of unicast messaging
+
+config MESSAGING_UNICAST_MSGSIZE
```

Comment:
Is there any limit for the message size? Can it exceed the MQ_MAXMSGSIZE?

Reply:
No.. It cannot exceed the MQ_MAXMSGSIZE. So I hide this config and set to default(MQ_MAXMSGSIZE). Thanks.

Reply:
Done.

Reply:
Done.